### DISCUSSION TITLE
Tests 4.1.7 and 4.1.8 are marked Manual but have audit scripts

### DISCUSSION THREAD

**Discussion by mrajashree** [aquasecurity/kube-bench#763] [Questions and Help] [answered]
The node.yaml in cis-1.6 has two tests [4.1.7](https://github.com/aquasecurity/kube-bench/blob/master/cfg/cis-1.6/node.yaml#L99) and [4.1.8](https://github.com/aquasecurity/kube-bench/blob/master/cfg/cis-1.6/node.yaml#L116) that are marked `(Manual)` but have audit scripts. I think that means they can be marked as `(Automated)`? Also these tests have `scored: false` although they were [scored:true in cis-1.5](https://github.com/aquasecurity/kube-bench/blob/master/cfg/cis-1.5/node.yaml#L122-L140)
I think they are marked scored:false because with the addition of CIS 1.6, [Manual/Automated translates to scored/not-scored](https://github.com/aquasecurity/kube-bench/discussions/664). If that's the case then they should be marked `(Automated)` and scored:true

**Comment by mozillazg** [comment, marked as answer]
> The node.yaml in cis-1.6 has two tests 4.1.7 and 4.1.8 that are marked (Manual) 

That's because the CIS Kubernetes Benchmark v1.6.0 mark them as `(Manual)` and the CIS Kubernetes Benchmark v1.5.1 mark them as `(Scored)`, we should follow the CIS and keep them as `(Manual)` in cis-1.6 and as `(Scored)` in cis-1.5.
